elements/pfe-readtime/demo/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+    <title>PatternFly Element | pfe-readtime Demo</title>
+
+    <noscript>
+      <link href="../../pfelement/dist/pfelement-noscript.min.css" rel="stylesheet">
+    </noscript>
+
+    <link href="../../pfelement/dist/pfelement.min.css" rel="stylesheet">
+
+    <!-- Stylesheets for testing light DOM styles.
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap-reboot.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/typebase.css/0.5.0/typebase.css">
+    -->
+
+    <link rel="stylesheet" href="../../pfe-styles/dist/pfe-base.css" />
+    <!-- <link rel="stylesheet" href="../../pfe-styles/dist/pfe-context.css" /> -->
+    <!-- <link rel="stylesheet" href="../../pfe-styles/dist/pfe-layouts.css" /> -->
+    <link rel="stylesheet" type="text/css" href="../../pfe-styles/dist/red-hat-font.min.css">
+
+    <link href="demo.css" rel="stylesheet">
+
+    <!-- uncomment the es5-adapter if you're using the umd version -->
+    <script src="/examples/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+
+    <!-- Use webcomponents-loader when you are adding support for more modern browsers -->
+    <!-- <script src="/examples/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script> -->
+
+    <!-- IE11 test: use bundle + require with umd files -->
+    <!-- Use webcomponents-bundle when supporting much older browsers like IE11 -->
+    <script src="/examples/node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+
+    <!-- Tests require pulling in the UMD version of the files -->
+    <script src="/examples/node_modules/requirejs/require.js"></script>
+
+    <script>require([
+      "../dist/pfe-readtime.umd.js"
+    ])</script>
+  </head>
+
+  <body unresolved>
+
+    <div data-background="gray">
+      <div class="container" data-padding="30">
+        <h1>&#x3C;pfe-readtime&#x3E;</h1>
+      </div>
+    </div>
+
+    <!-- Example 1 -->
+    <div class="container" data-padding="30">
+
+      <pfe-readtime for="readtime1" id="pfeReadtime" hidden test>
+        <span slot="read-string">%t-minute readtime</span>
```

Comment:
Can we add a second demo with this:
```suggestion
        <span slot="read-string">Minutes to read: %t or so</span>
```
to demonstrate that it is `%t` which is being replaced and not the whole string?